### DISCUSSION TITLE
Activate scene from Character Sheet

### DISCUSSION THREAD

**Discussion by SwedishRabbit** [fvtt-fria-ligan/forbidden-lands-foundry-vtt#328] [Ideas]
Is it possible to add a button on the upper corner of the character sheet(s) to "show it" as a scene for everybody. Even the player who owns the character sheet could do this.

The benefit would be to show when having a question or tutorial, or if someone wants to temporary share his sheet when making a dice roll.

It could have a pop up GM approval "Ok" before it gets into an activated scene.

**Comment by aMediocreDad** [comment]
Hey good idea! But this falls completely out of the scope of the system and should be a system agnostic module. Definitely sounds like a cool thing, and could also be someone else have thought of it and implemented it already as a module.

Look around in the modules list on Foundry and perhaps ask around in the module discussion channels on Discord.

**Reply by SwedishRabbit** [reply]
Will do! Thanks!
